src/CattyTests/Bricks/GlideToBrickTests.swift

```diff
@@ -37,4 +37,40 @@ final class GlideToBrickTests: XCTestCase {
         XCTAssertEqual(brick.xDestination, brick.formula(forLineNumber: 1, andParameterNumber: 0))
         XCTAssertEqual(brick.yDestination, brick.formula(forLineNumber: 1, andParameterNumber: 1))
     }
+
+    func testMutableCopy() {
+        let brick = GlideToBrick()
+
+        brick.durationInSeconds = Formula(double: 1)
+        brick.xDestination = Formula(double: 1)
+        brick.yDestination = Formula(double: 1)
+
+        let copiedBrick: GlideToBrick = brick.mutableCopy(with: CBMutableCopyContext(), andErrorReporting: true) as! GlideToBrick
+
+        XCTAssertTrue(brick.isEqual(to: copiedBrick), "GlideToBricks are not equal")
```

Comment:
You tests are very readable. Please remove all messages in the assert statements since they do not deliver any additional information (your test code explains already everything).